src/Test.UnitTests.Sarif/Visitors/SuppressVisitorTests.cs

```diff
@@ -19,88 +20,108 @@ public class SuppressVisitorTests
+                    Guids = new List<string>()
                 },
                 new
                 {
                     Alias = "some alias",
                     Justification = "some suppress justification",
-                    Guids = true,
+                    Uuids = true,
                     Timestamps = true,
                     ExpiryInDays = 1,
-                    SuppressionStatus = SuppressionStatus.UnderReview
+                    SuppressionStatus = SuppressionStatus.UnderReview,
+                    Guids = new List<string>()
+                },
+                new
+                {
+                    Alias = "some alias",
+                    Justification = "some suppress justification",
+                    Uuids = true,
+                    Timestamps = true,
+                    ExpiryInDays = 1,
+                    SuppressionStatus = SuppressionStatus.Accepted,
+                    Guids = guids.ToList()
                 },
             };
 
             foreach (var testCase in testCases)
             {
                 VerifySuppressVisitor(testCase.Alias,
                                       testCase.Justification,
-                                      testCase.Guids,
+                                      testCase.Uuids,
                                       testCase.Timestamps,
                                       testCase.ExpiryInDays,
-                                      testCase.SuppressionStatus);
+                                      testCase.SuppressionStatus,
+                                      testCase.Guids);
             }
         }
 
         private static void VerifySuppressVisitor(string alias,
                                                   string justification,
-                                                  bool guids,
+                                                  bool uuids,
                                                   bool timestamps,
                                                   int expiryInDays,
-                                                  SuppressionStatus suppressionStatus)
+                                                  SuppressionStatus suppressionStatus,
+                                                  IEnumerable<string> guids)
         {
             var visitor = new SuppressVisitor(justification,
                                               alias,
-                                              guids,
+                                              uuids,
                                               timestamps,
                                               expiryInDays,
-                                              suppressionStatus);
+                                              suppressionStatus, 
+                                              guids);
 
             var random = new Random();
             SarifLog current = RandomSarifLogGenerator.GenerateSarifLogWithRuns(random, runCount: 1, resultCount: 1);
```

Comment:
>current [](http://example.com/codeflow?start=21&length=7)

set the result's guid to using value in `guids` list. then verify if right result is suppressed if `guids` is not empty, so it can cover the changes you made in SuppressVisitor.cs